sox.features.js

```diff
@@ -1254,11 +1254,22 @@
         });
       }
 
-      addLabels();
+      function addQuestionStateInReview() {
+        sox.warn("called")
+        const $anchor = $('.summary h2 a');
+        const $aText = $anchor.html();
+        sox.warn($aText)
+        if ($('[class^="standOutDupeCloseMigrated"]').length || !$anchor.length || !$aText.match(/closed|duplicate/)) return;
 
+        $anchor.html($aText.replace(/ \[(closed|duplicate)\]/, ''));
+        $anchor.after(`&nbsp;<span class="standOutDupeCloseMigrated-${$aText.match(/closed|duplicate/)[0]}">${$aText.match(/closed|duplicate/)[0]}</span></a>`);
+      }
+
+      addLabels();
       const targetMainPage = document.getElementById('question-mini-list');
       const targetQuestionsPage = document.getElementById('questions');
       sox.helpers.observe([targetMainPage, targetQuestionsPage], '#questions, #question-mini-list', addLabels);
+      $(document).on('sox-new-review-post-appeared', addQuestionStateInReview);
```

Comment:
Could we modify `addLabels` to work for this case too? the advantage of using that function is that it fetches some extra data from the API like when it was closed and by who

I think it might be possible to add another thing like `const questionSummaries = ...` but instead be like `const reviewPosts = ...` and loop through them and push them to the `questions` array with the data we need

Reply:
Although I see that we didn't do this before, so if it looks like it's harder than what I said above let me know and that can be a future change! :)